src/test/java/org/fundacionjala/trello/runner/Runner.java

```diff
@@ -22,6 +23,13 @@
     @BeforeTest
     public void beforeAllScenarios() {
         // Code executed before features execution.
+        System.setProperty("dataproviderthreadcount", "3");
```

Comment:
I suggest you get this value (`"3"`) from Environment properties.

Reply:
Good observation, working on it!